service_container/autowiring.rst

```diff
@@ -427,12 +465,23 @@ that alias:
         $container->autowire(TwitterClient::class)
             //->setArgument('$transformer', new Reference(UppercaseTransformer::class))
         ;
+        $container->registerAliasForArgument(
+            TransformerInterface::class,
+            MastodonClient::class,
```

Comment:
It's not `MastodonClient` but `UppercaseTransformer`.

Reply:
Good catch lol!